docs/docs/sourcing-from-woocommerce.md

```diff
@@ -0,0 +1,90 @@
+    api: 'wordpress.domain',
+    // true if using https. false if nah.
+    https: false,
+    api_keys: {
+      consumer_key: `your key`,
+      consumer_secret: `your secret`,
+    },
+    // Array of strings with fields you'd like to create nodes for...
+    fields: ['products', 'products/categories', 'products/attributes'],
+  }
+},
+```
+
+The consumer key and secret come from WooCommerce. From your dashboard, go to WooCommerce > Settings > Advanced > REST API and then add a key. You can then copy and paste both the key and secret. Make sure to store any sensitive information in [environment variables](/docs/environment-variables/)!
+
+There are other options which may make more sense for your project. For example, you may want to source more general site data like posts and pages from WordPress as well.
+
+### Pending Changes
+
+Note that `gatsby-source-wordpress` is undergoing a major rewrite at the time of writing. This new major version of the official source plugin is likely to change the optimal solution for using WooCommerce with Gatsby. For now, new projects should start with [`gatsby-source-graphql`](/packages/gatsby-source-graphql/) rather than `gatsby-source-wordpress` to ease the transition. If you’re interested in the current progress of the plugin, you can [check out the working alpha version](https://github.com/gatsbyjs/gatsby/issues/19292#issuecomment-587946239). **Once this work is done, the [`wp-graphql-woocommerce` plugin](https://github.com/wp-graphql/wp-graphql-woocommerce) will be the recommended option.**
+
+If you decide to go with this more experimental route, you'll need to install and activate the [WPGraphQL](https://www.wpgraphql.com/) and [wp-graphql-woocommerce](https://github.com/wp-graphql/wp-graphql-woocommerce) plugins on your WordPress site as well.
+
+The WPGraphQL plugin is also undergoing significant changes. It has not yet hit v1 and may therefore introduce [breaking changes](https://docs.wpgraphql.com/getting-started/install-and-activate/#breaking-change-notice) as it continues to progress through development. It can be used in production but keep this in mind when using the plugin!
+
+## Adding products
+
+WooCommerce products are the core of your WooCommerce site, so you’ll want to add some to your store. When you’re first getting started, there’s no need to dump absolutely everything you want to sell into your store. Begin with a few products that you can use to verify that everything is hooked up correctly. You can also [import products from a CSV file](https://docs.woocommerce.com/document/product-csv-importer-exporter/). 
+
+You should see a prompt to create your first product after completing the WooCommerce setup wizard. Doing so at this time will trigger a guided tour of the new product form. Walk through this process for each product you'd like to add to your store.
+
+Once this is done, you’ll be able to use GraphQL to query for your products:
+
+```graphql
```

Comment:
Is there a file we can recommend here? Or is it just whatever page/component you'll be displaying this info on?

Reply:
Whatever page/component! I started to suggest a specific page but then wandered down a rabbit hole towards building out a whole site vs showing people how to get at the data. That's one reason I included the theme in "Other resources". Might be worth linking to the live example they have but I thought bringing that in here might be confusing since I'm not listing the theme as a prerequisite.

Reply:
Gotcha. Might be worth noting that in text? "...to query for your products by adding this query to the appropriate component"